src/QCheck.mli

```diff
@@ -170,8 +170,8 @@ module Gen : sig
   val join : 'a t t -> 'a t
   (** @since NEXT_RELEASE *)
 
-  val sized : 'a sized -> 'a t
-
+  val sized : 'a sized -> 'a t (** Create a generator from a size-bounded generator *)
```

Comment:
I think this could be more detailed (i.e. "first it generates a random size and use it on the sized generator" or sth like this)"